packages/amplify-cli/src/attach-backend.ts

```diff
@@ -100,7 +100,10 @@ const onSuccess = async (context: $TSContext): Promise<void> => {
   } else if (stateManager.currentMetaFileExists()) {
     await initializeEnv(context, stateManager.getCurrentMeta());
   }
-
+  // move Hooks folder from backup to original amplify folder
+  const HooksDirPath = pathManager.getHooksDirPath(projectPath);
+  const HooksBackupDirPath = path.join(backupAmplifyDirPath, 'hooks');
+  fs.moveSync(HooksBackupDirPath, HooksDirPath, { overwrite: true });
```

Comment:
1. is it possible that `hooks` doesn't exist at all?
2. Should we aways overwrite if it exists at the destination ?

Reply:
1. Nice catch! , I can include a check but usually we  add a hooks folder during init .
2. Yes , There us no hooks present at destination as we already put that in backup amplify holder while generating backend.